llvm/include/llvm/CodeGen/GlobalISel/GIMatchTableExecutor.h

```diff
@@ -635,8 +636,12 @@ class GIMatchTableExecutor {
 
   bool shouldOptForSize(const MachineFunction *MF) const {
     const auto &F = MF->getFunction();
-    return F.hasOptSize() || F.hasMinSize() ||
```

Comment:
Lost the hasMinSize check?

Reply:
As mentioned above (but got lost in the changes), `F.hasOptSize()` already calls `hasMinSize()`.
https://github.com/llvm/llvm-project/blob/927af63fddb8e34f23b2974f812156767988ec5f/llvm/include/llvm/IR/Function.h#L707-L709